tests/installation/bootloader_s390.pm

```diff
@@ -94,13 +94,16 @@ ENTER
 ENTER
 EO_frickin_boot_parms
 
+    # Define memory to behave the same way as other archs
+    # and to have the same configuration through all s390 guests
+    # convert MB -> GB, it's not like perl can't calculate
+    my $ram = get_var('QEMURAM', 1024) / 1024;
```

Comment:
fails if e.g. QEMURAM=512 since you can't do "Define storage" with decimal values...

`DEFINE STORAGE 0.5G`
`HCPZPM003E Invalid option - 0.5G`